### PR TITLE
[Merged by Bors] - Mention search filters in CONTRIBUTING.md

### LINKED ISSUE #6221: Add useful search filters to CONTRIBUTING.md
>  Useful search filters for reviewing:
> 
> - PRs that could use more reviews: https://github.com/bevyengine/bevy/pulls?q=is%3Aopen+is%3Apr+-label%3AS-Ready-For-Final-Review+-draft%3A%3Atrue+-label%3AS-Needs-RFC+-reviewed-by%3A%40me+-author%3A%40me
> - PRs that are ready for maintainers to merge without consultation: https://github.com/bevyengine/bevy/pulls?q=is%3Aopen+is%3Apr+label%3AS-Ready-For-Final-Review+-label%3AS-Controversial+-label%3AS-Blocked+-label%3AS-Adopt-Me+
> - PRs that need final input from @cart: https://github.com/bevyengine/bevy/pulls?q=is%3Aopen+is%3Apr+label%3AS-Ready-For-Final-Review+label%3AS-Controversial+
> 
> You can add more filters to these searches: you may find it helpful to e.g. specify by A-UI to only identify PRs that need review that touch on bevy_ui.
> 
> If your PR is not yet ready for review (or there are serious outstanding issues that you need to to fix) please put it into draft mode so that these search filters work correctly! 

This should be documented in CONTRIBUTING.md, rather than living in a pinned message. Ideally these are spread into the appropriate sections, rather than copied in verbatim.

### PR DESCRIPTION
**Pull request by Ptipiak**
* Adding a new section concerning the maintainers of the repo

# Objective

- Adding a few helpful links in the CONTRIBUTING.md files
- Fixes #6221 

## Solution

- Modifying CONTRIBUTING.md
- Adding a new section dedicated to maintainers  in CONTRIBUTING.md

---

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -274,6 +275,13 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### Maintaining code
+
+This section is more dedicated to newly added maintainers of the repo, this new status allow you to merge pull request, you'll find multiple tips and links to where look for those requests in need of merging, and the one which need a final approval and review from @cart.
```
Comment (alice-i-cecile) [inline CONTRIBUTING.md:280]: This sentence is very long :p Can we split this up?

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Thanks for doing this :) Just needs a bit of wording help down below.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Ptipiak) [reply CONTRIBUTING.md]: I splitted up the sentence in three parts, let me now if it doesn't fit with what you had in mind

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -274,6 +275,17 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### Maintaining code
+
+This section is more dedicated to newly added maintainers of the repo.
```
Comment (alice-i-cecile) [inline CONTRIBUTING.md:280]:
```suggestion
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -274,6 +275,17 @@ By giving feedback on this work (and related supporting work), you can help us m
...
+This new status allow you to merge pull request, this section aim to provide tips and helpful links to help you maintain the project.
```
Comment (alice-i-cecile) [inline CONTRIBUTING.md:281]:
```suggestion
Maintainers can merge uncontroversial pull requests  that have at least two approvals (or at least one for trivial changes).
```

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Definitely better than before. I left some more wording suggestions for you: I think that this information is useful for everyone to be able to see, not just maintainers.

**Review by alice-i-cecile — APPROVED** [review]

**Review by Nilirad — APPROVED** [review]
Looks good. Maybe reference-style links may help plain text legibility but that's just a personal preference.

**Comment by alice-i-cecile** [comment]
Thanks!

bors r+

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -274,6 +275,16 @@ By giving feedback on this work (and related supporting work), you can help us m
 Finally, if nothing brings you more satisfaction than seeing every last issue labeled and all resolved issues closed, feel free to message @cart for a Bevy org role to help us keep things tidy.
 As discussed in [*How we're organized*](#how-were-organized), this role only requires good faith and a basic understanding of our development process.
 
+### Maintaining code
+
+Maintainers can merge uncontroversial pull requests  that have at least two approvals (or at least one for trivial changes).
```
Comment (mockersf) [inline CONTRIBUTING.md:280]:
```suggestion
Maintainers can merge uncontroversial pull requests that have at least two approvals (or at least one for trivial changes).
```

**Comment by Ptipiak** [comment]
Thanks, for some reason my `markdownlint` wasn't picking the extra white space locally

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -274,6 +275,16 @@ By giving feedback on this work (and related supporting work), you can help us m
...
+### Maintaining code
+
+Maintainers can merge uncontroversial pull requests that have at least two approvals (or at least one for trivial changes).
+
+These search filters show the requests that can be merged by maintainers, and those which need a final approval from @cart.
+
+1. Pulls requests which are ready for maintainers to merge without consultation: [requests to pull](https://github.com/bevyengine/bevy/pulls?q=is%3Aopen+is%3Apr+label%3AS-Ready-For-Final-Review+-label%3AS-Controversial+-label%3AS-Blocked+-label%3AS-Adopt-Me+)
+2. Pull requests which need final input from @cart: [requests to verify](https://github.com/bevyengine/bevy/pulls?q=is%3Aopen+is%3Apr+label%3AS-Ready-For-Final-Review+label%3AS-Controversial+)
+
+
```
Comment (mockersf) [inline CONTRIBUTING.md:287]:
```suggestion
```

**Review by alice-i-cecile — APPROVED** [review]
bors retry

**Pull request closed without merging**